src/init.cpp

```diff
@@ -1553,7 +1553,7 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     }
 
     if (chainparams.GetConsensus().vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout != 0) {
-        // Only advertize witness capabilities if they have a reasonable start time.
```

Comment:
@rebroad Do you know in what countries? As I've understood it "advertise" is the form used in both US and UK, but there are more English speaking countries than that obviously :-)